packages/paste-website/src/pages/patterns/filter/index.mdx

```diff
@@ -0,0 +1,632 @@
+          return (
+            <PopoverContainer key={pill} state={popover}>
+              <PopoverButton
+                variant="reset"
+                size="reset"
+                borderRadius="borderRadiusPill"
+              >
+                <FormPill
+                  {...pillState}
+                  selected={isSelected}
+                  onDismiss={
+                    // Remove the filter from the selected filters
+                  }
+                >
+                  {!isSelected ? <PlusIcon decorative /> : null}
+                  <FilterPillView label={filterMap[pill].label} selectedType={isSelected ? pill : null} selectedValue={value} />
+                </FormPill>
+              </PopoverButton>
+              <Popover aria-label={pill} width="size40">
+                <FilterComponent value={value} onApply={onApply} popover={popover} onRemove={onRemove} /> // Create different filter components
+              </Popover>
+            </PopoverContainer>
+          );
+        })}
+      </FormPillGroup>
+      <Box display="flex" justifyContent="space-between" alignItems="center" columnGap="space30" marginTop="space50">
+        <Box display="flex" columnGap="space30">
+          <DetailText marginTop="space0">
+            <Text as="span" color="colorTextWeak" fontSize="fontSize30">
+              {filteredTableData.length} result{filteredTableData.length !== 1 && "s"}
+            </Text>
+          </DetailText>
+          {filteredTableData.length !== data.length ? (
+            <Button variant="link" onClick={handleClearAll}>
+              Clear all
+            </Button>
+          ) : null}
+        </Box>
+      </Box>
+      <DataGrid data={filteredTableData} />
+    </>
+  )
+}`}
+/>
+
+#### Surfacing results
+<UnorderedList>
+  <ListItem>When a filter value is selected by the user, the corresponding Form Pill will enter a selected state.</ListItem>
+  <ListItem>When a single filter value is selected, the filter updates to display the name of the selected filter.</ListItem>
+  <ListItem>When two or more filter values are selected per filter category, the total number of selected filters will be displayed in a counter badge.</ListItem>
+  <ListItem>In some scenarios, it is necessary to have filter values pre-selected by default to display results. Since the selection was not made by the user, the Form Pill will remain in its default state, displaying the pre-selected filter value.</ListItem>
+</UnorderedList>
+
+ <ResponsiveImage src={FilterBarImage} alt="The filter bar has different states based on interaction" />
+
+<DoDont>
+  <Do title="Do" body="Arrange filters in a way that mirrors how users think about and interact with the data." />
+  <Dont title="Don't" body="The filter bar is designed to provide prioritized, essential filters for quick application. Avoid adding too many filters, as this can clutter the screen and make it harder for users to find relevant information." />
```

Comment:
**todo:** Update this text: Don't overload the filter bar with too many filters. It's designed for prioritized, essential filters to keep the screen uncluttered and easy to navigate.